src/components/ProductList.tsx

```diff
@@ -0,0 +1,41 @@
+import React, { useEffect, useState } from 'react';
+
+interface Product {
+  id: number;
+  image: string;
+  title: string;
+  price: number;
+}
+
+function ProductList() {
```

Comment:
Is this component old? It's not imported anywhere

Reply:
Yes it has been replaced by the categories/product grid page but I still use it now as a placeholder for the landing page